src/Makefile.am

```diff
@@ -142,6 +144,15 @@ AM_V_SED = $(am__v_SED_@AM_V@)
 am__v_SED_ = $(am__v_SED_@AM_DEFAULT_V@)
 am__v_SED_0 = @echo "  SED     " $@;
 
+build_info.h: $(srcdir)/../.git/HEAD $(srcdir)/../.git/index $(top_builddir)/config.status
```

Comment:
Oh, crud. This will probably break our release tarballs, since they aren't git repositories...  Maybe I should add a configure test to disable the git output instead of including git as a make dependence.